package.json

```diff
@@ -110,10 +110,10 @@
     "re-reselect": "^3.4.0",
     "react": "^16.13.1",
     "react-i18next": "11.7.0",
-    "react-native": "0.61.2",
+    "react-native": "0.63.2",
```

Comment:
This is not obvious because React Native uses the minor part of their version as a major, introducing invasive changes, and nothing really blocks you from doing so, but you can't just bump RN minors as any other package.
You need to check what changes had been made for everything React Native outside of `node_modules` and apply them to our repo. Thankfully they now provide a handy upgrade helper, listing them all from any specific version to any other : https://react-native-community.github.io/upgrade-helper/?from=0.61.2&to=0.63.2

At a glance, you'll see that 20+ files changed from our RN version to the one you're targeting, versus only 8 in this PR, so we're missing stuff.

Reply:
Yes i've seen that one i just didn't need to change anything on android was working as is, so i skiped on the IOS part my bad :sweat_smile: I'll try and look for a solution with someone that can build on IOS since i can't.